operations/jsonnet/single-binary/tempo.libsonnet

```diff
@@ -43,7 +43,9 @@
     container.withVolumeMounts([
       volumeMount.new(tempo_config_volume, '/conf'),
       volumeMount.new(tempo_data_volume, '/var/tempo'),
-    ]),
+    ]) +
+    k.util.resourcesRequests('3', '3Gi') +
```

Comment:
i'd rather not set limits/requests in the vendored jsonnet. these values are so specific to a given installation that i think they should always be set by the caller.

Reply:
heh, nevermind, i see we were doing it before :P